pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -813,6 +813,16 @@ func (og *operationGenerator) GenerateUnmountVolumeFunc(
 			return volumeToUnmount.GenerateError("UnmountVolume.TearDown failed", unmountErr)
 		}
 
+		// Successful unmount event is useful for user debugging
+		simpleMsg, _ := volumeToUnmount.GenerateMsg("UnmountVolume.TearDown succeeded", "")
+
+		if volumeToUnmount.VolumeSpec != nil && volumeToUnmount.VolumeSpec.PersistentVolume != nil && volumeToUnmount.VolumeSpec.PersistentVolume.Spec.ClaimRef != nil {
+			pod, _ := og.kubeClient.CoreV1().Pods(volumeToUnmount.VolumeSpec.PersistentVolume.Spec.ClaimRef.Namespace).Get(string(volumeToUnmount.PodName), metav1.GetOptions{})
```

Comment:
@jingxu97 @msau42 I don't think `kubeclient get pod` is good solution here. But struct `volumeToUnmount` doesn't contain the point of pod.
The same is the struct `AttachedVolume` which is used for detach volume.
Do you have better idea ?